automatic/intunewinapputil/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Win32 Content Prep Tool
```

Comment:
```suggestion
# [intunewinapputil](https://community.chocolatey.org/packages/intunewinapputil)
```

This should be the same as the title of the package, as well as a link to the package

Reply:
I've updated this.